less/navigation.less

```diff
@@ -1 +1,38 @@
-// Navigation Styles here
\ No newline at end of file
+@import "variables.less";
```

Comment:
if you import your less files correctly in index.less, you don't need these imports.